File: sumPairs.js
pairs = []
function sumPairs(array, sum){
  for(let i = 0; i < array.length - 1; i++){
    for(let k = i+1; k < array.length; k++){
    if(array[i] + array[k] === sum){
        pairs.push([array[i], array[k]])
    } else {
        return 'unable to find pairs'
}
  }
}
return pairs

}

sumPairs([1,2,3,4,5], 7)

File: sum_pairs.py
pairs = []
def sum_pairs(array, sum):
  for i in range(len(array)-1):
        for k in range(i+1, len(array)):
            if array[i] + array[k] == sum:
                pairs.append([array[i], array[k]])
            else:
                print('unable to find pairs')
  print(pairs)


sum_pairs([1,2,3,4,5], 7)
